k8s/kops-weave/weave.yml

```diff
@@ -203,7 +204,7 @@ items:
                 timeoutSeconds: 5
               resources:
                 requests:
-                  cpu: 1000m
+                  cpu: 100m
```

Comment:
Setting a lower request for CPU, so that we have a higher chance of scheduling weave on a given host.

Reply:
Ultimately this won't matter much when we handle autoscaling in Testground.